pyproject.toml

```diff
@@ -142,7 +143,6 @@ max-line-length = "120"
 ignored-argument-names = "args|kwargs|_|__"
 
 [tool.pylint.BASIC]
-good-names = "_,__,i,e,k,v,fn,fs,get,post,put,patch,delete,route,asgi,websocket,Dependency,Body,Parameter,HandlerType,ScopeType,Auth,User,it"
```

Comment:
removed because this is becoming very long, and frankly this inspection is a nuisance.

Reply:
sgtm.